index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+   <link rel="stylesheet" href="style.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+   <header>
+      <div class="logo"><a href="#">My Logo</a></div>
+      <nav>
+         <div class="hamburger-menu"><i class="fa-solid fa-bars"></i></div>
```

Comment:
- [ ] Nice work making the changes suggested by the previous reviewers :clap: However your hamburger menu icon is overlap and not on the same line with the logo as in the Figma design. Kindly fix it
### Your design :point_down:  
![Screenshot from 2023-06-02 16-15-49](https://github.com/Isaac-ej/Portfolio_Website/assets/91142432/0cae08f2-d470-40a9-8aaf-565367aced45)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-05-29 09-31-09](https://github.com/Isaac-ej/Portfolio_Website/assets/91142432/b4c680c5-f0e8-4b7d-96c6-cbdb9124c780)